### DISCUSSION TITLE
Add language

### DISCUSSION THREAD

**Discussion by Marsaarz** [Quplet/Qu-Enchantments#8] [Ideas]
Sorry for having no experience for creating full_requests.I made a Chinese language file for the mod.Could you please put it in your 1.19 verson? :)
(Sorry again for my broken English)
[zh_cn.zip](https://github.com/Quplet/Qu-Enchantments/files/9466725/zh_cn.zip)

**Comment by Quplet** [comment]
Sure, I'll add it once I get the chance. Probably will push it out tomorrow night.

**Reply by Marsaarz** [reply]
emm。。。There is a small problem that I typed a wrong word.So sorry about troubling you again for such a piece of biscuit.
Have a good day ：）
[zh_cn.zip](https://github.com/Quplet/Qu-Enchantments/files/9469556/zh_cn.zip)
